active/website.md

```diff
@@ -0,0 +1,62 @@
+# Website Working Group
+
+## Scope of responsibilities
+
+This is a replacement of the current maintainer team @django/djangoproject-com-maintainters. The team will own maintenance of the website codebase, and liaise with the @django/ops-team for production infrastructure considerations.
+
+The duties of the working group are:
+- Introduce new features on the website
+- Maintain and monitor the website 
+- Ensure that information on the website is accurate 
+- Help to make the website accessible to all
+
```

Comment:
We also run or support djangoproject.com sessions in DjangoCon sprints.
https://github.com/django/djangoproject.com/pulls?q=is%3Apr+is%3Aclosed+label%3A%22DjangoCon+%F0%9F%A6%84%22

Reply:
@cgl thanks

Reply:
@thibaudcolas any objections to this?

Reply:
No?